testng/src/main/java/io/cucumber/testng/TestNGCucumberRunner.java

```diff
@@ -62,6 +63,7 @@
     private final RuntimeOptions runtimeOptions;
     private final Plugins plugins;
     private final FeaturePathFeatureSupplier featureSupplier;
+    private final List<Feature> features = new ArrayList<>();
```

Comment:
This should be initialized as null. There is a difference between not initialized and no feature found.

Reply:
Updated accordingly